src/styles/todo.scss

```diff
@@ -73,6 +73,7 @@
   &__title-field {
     width: 100%;
     padding: 11px 14px;
+    box-sizing: border-box;
```

Comment:
You can avoid using box-sizing on every field by setting it to all html elements
```css
* {
  box-sizing: border-box;
}
```